src/org/zaproxy/zap/extension/pscanrulesBeta/CSRFCountermeasures.java

```diff
@@ -216,45 +216,47 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 					evidence = formElement.getFirstElement().getStartTag().toString();
 				}
 				numberOfFormsPassed++;
```

Comment:
This should be incremented always if we want a consistent form numbering? (See other info in last comment, the second alert mentions "Form 2" but it's actually the third form of the page.)